src/main/kotlin/no/nav/familie/ef/iverksett/vedtakstatistikk/VedtakstatistikkKafkaProducer.kt

```diff
@@ -1,21 +1,22 @@
 package no.nav.familie.ef.iverksett.vedtakstatistikk
 
+import no.nav.familie.eksterne.kontrakter.ef.BehandlingDVH
 import org.slf4j.LoggerFactory
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.kafka.core.KafkaTemplate
 import org.springframework.stereotype.Service
 
 @Service
 class VedtakstatistikkKafkaProducer(
-    private val kafkaTemplate: KafkaTemplate<String, String>
+    private val kafkaTemplate: KafkaTemplate<String, BehandlingDVH>
```

Comment:
Det er nok enklest å lese for datavarehus å consume dersom vi gjør om til json-string i stedet, altså at både key og value er av type String